src/drivers/distance_sensor/tfmini/tfmini_main.cpp

```diff
@@ -46,6 +46,7 @@ TFMINI	*g_dev{nullptr};
 int start(const char *port, uint8_t rotation);
 int status();
 int stop();
+int command(uint8_t *command, uint8_t framelen);
```

Comment:
@amoizard , could you alphabetize the order of these method declarations? Thanks!

Reply:
> @amoizard , could you alphabetize the order of these method declarations? Thanks!

I just moved the declaration of "command" method.
I have also removed a duplicate description of "tfmini status" command..
And I have also re-indented the end of tfmini_main.cpp where ./Tools/astyle/fix_code_style.sh is not working.